app/visits/delete/template.hbs

```diff
@@ -5,7 +5,7 @@
     updateButtonText=updateButtonText }}
   <div class="alert alert-danger">
     <span class="glyphicon glyphicon-warning-sign"></span>
-    {{t 'visits.messages.delete'}}
+    {{t 'messages.delete_singular' name='visit'}}
```

Comment:
This code is passing a non localized string when it should be passing in a localized string or it should use the name of the item being deleted.